status/reporter_github.go

```diff
@@ -52,14 +49,14 @@ type StatusUpdater interface {
 	// Authentication of client
 	Authenticate(ctx context.Context, snapshot *applicationapiv1alpha1.Snapshot) error
 	// Update status of PR
-	UpdateStatus(ctx context.Context, integrationTestStatusDetail intgteststat.IntegrationTestStatusDetail) error
+	UpdateStatus(ctx context.Context, report TestReport) error
 }
 
 // CheckRunStatusUpdater updates PR status using CheckRuns (when application integration is enabled in repo)
 type CheckRunStatusUpdater struct {
 	ghClient          github.ClientInterface
 	k8sClient         client.Client
-	logger            *helpers.IntegrationLogger
```

Comment:
Just curious about this change, is it because we don't gain anything by using `IntegrationLogger` or is there something more behind it?

Reply:
I didn't like that helpers are crossimported everywhere and we don't need anything from IntegrationLogger here. 
Application name is already in underlying logger

Reply:
Fair enough, thanks for clarifying.